### PR TITLE
add k8s support

### PR DESCRIPTION
**Pull request by WPH95**
Signed-off-by: wph95 <wph657856467@gmail.com>

### CODE REVIEW

File: k8s/30dashbase-table.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  template:
+    metadata:
+      labels:
+        app: dashbase-table
+    spec:
+      imagePullSecrets:
+      - name: docker-image-secret
+      containers:
+      - name: dashbase-table-nginx
+        image: dashbase/table:1.0.0-rc14
```
Comment (javasoze) [inline k8s/30dashbase-table.yml]: use rc15

File: k8s/31dashbase-api.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    metadata:
+      labels:
+        app: dashbase-api
+    spec:
+      imagePullSecrets:
+      - name: docker-image-secret
+      containers:
+      - name: quickstart-api
+        image: dashbase/api:1.0.0-rc14
+        command: ['/dashbase-api-1.0.0-rc14/bin/dashbase-api', 'server', '/config/api.yml']
```
Comment (javasoze) [inline k8s/31dashbase-api.yml]: rc15

File: k8s/32dashbase-web.yml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  template:
+    metadata:
+      labels:
+        app: dashbase-web
+    spec:
+      imagePullSecrets:
+      - name: docker-image-secret
+      containers:
+      - name: quickstart-web
+        image: dashbase/web:1.0.0-rc14
```
Comment (javasoze) [inline k8s/32dashbase-web.yml]: rc15

**Comment by khou** [comment]
Please reopen on deployment repo

**Pull request closed without merging**